sdk/storage/storage-blob/src/Pipeline.ts

```diff
@@ -200,7 +203,14 @@ export function newPipeline(
     new BrowserPolicyFactory(),
     deserializationPolicy(), // Default deserializationPolicy is provided by protocol layer
     new RetryPolicyFactory(pipelineOptions.retryOptions),
-    new LoggingPolicyFactory()
+    logPolicy(
+      logger.info, {
+        allowedHeaderNames: [
+          "x-ms-blob-append-offset",
+          "x-ms-blob-committed-block-count"
```

Comment:
I saw there's another PR with a set of default headers to enable.  Is this whitelist only for headers specific to Storage and to be determined by Storage team?

Reply:
That PR was for the whitelist that works across all SDK libraries, the headers specified here are whitelisted in addition to the default list.

Reply:
Prefer to move constant lists into constants.ts

---
In reply to: [336269928](https://github.com/Azure/azure-sdk-for-js/pull/5636#discussion_r336269928) [](ancestors = 336269928)